src/VueProgram.ts

```diff
@@ -40,6 +40,7 @@ export class VueProgram {
     );
 
     parsed.options.allowNonTsExtensions = true;
+    parsed.options = { ...parsed.options, ...compilerOptions };
```

Comment:
I'm not sure if we can use object spread as it won't work on node 6 (and fork-ts-checker supports node 6). https://node.green/#ES2018-features-object-rest-spread-properties-object-spread-properties

Can we switch to Object.assign please?

Reply:
Again - fine with this staying as is if TypeScript transpiles us to glory! 😄